scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java

```diff
@@ -157,47 +159,47 @@ protected static void start(CommandLine commandLine) throws Exception {
         hsqldbServer = new SchedulerHsqldbStarter();
         hsqldbServer.startIfNeeded();
 
-        String rmUrl = getRmUrl(commandLine);
+        rmURL = getRmUrl(commandLine);
+        //        rmURL = rmUrl;
         setCleanDatabaseProperties(commandLine);
         setCleanNodesourcesProperty(commandLine);
 
-        rmUrl = connectToOrStartResourceManager(commandLine, rmUrl);
+        rmURL = connectToOrStartResourceManager(commandLine, rmURL);
 
         if (commandLine.hasOption("rm-only")) {
             return;
         }
 
-        SchedulerAuthenticationInterface schedulerAuthenticationInterface = startScheduler(commandLine, rmUrl);
-
-        schedAuthInter = schedulerAuthenticationInterface;
-        rmURL = rmUrl;
+        schedulerURL = getSchedulerUrl(commandLine);
+        if (!commandLine.hasOption("scheduler-url")) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "scheduler-url" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)